src/Analyzers/CSharp/Tests/InlineDeclaration/CSharpInlineDeclarationTests.cs

```diff
@@ -23,7 +23,7 @@ internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProvider
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
         public async Task InlineVariable1()
         {
-            await TestInRegularAndScriptAsync(
+            await TestInRegularAndScript1Async(
```

Comment:
Is it possible to rename `TestInRegularAndScript1Async` to `TestInRegularAndScriptAsync`, so it is just an overload? I always wonder why we have `TestInRegularAndScript1Async` method...

Reply:
not currently because it will be ambiguous.  once TestInRegularAndScriptAsync is entirely gone, we can rename TestInRegularAndScript1Async to TestInRegularAndScriptAsync